go-controller/pkg/ovn/master_test.go

```diff
@@ -666,13 +691,9 @@ var _ = Describe("Gateway Init Operations", func() {
 		app = cli.NewApp()
 		app.Name = "test"
 		app.Flags = config.Flags
-
-		stopChan = make(chan struct{})
```

Comment:
Won't this mean stopChan will be nil and any attempt to use it will fail?

Reply:
it was inadvertent.